contracts/oracles/ChainlinkOracle.sol

```diff
@@ -5,7 +5,7 @@ import "@chainlink/contracts/src/v0.6/interfaces/AggregatorInterface.sol";
 
 contract ChainlinkOracle is IOracle{
 
-    address public oracle;
+    AggregatorInterface public oracle;
```

Comment:
`AggregatorInterface` is actually deprecated. For closer to real-time updates moving forward, we should use `AggregatorV3Interface`.